### ISSUE TITLE
Accessibility warning and app freeze

### ISSUE THREAD

**Issue by LIttleAncientForestKami** [patiem/ships_JA#49]
Replication scenario:

1.  git clone git@github.com:patiem/ships_JA.git test
2.  cd test/scripts 
3. ./launchClient.sh localhost
4. App launches, first screen is partially or fully rendered and completely unresponsive (frozen) - I cannot enter my name, I cannot press next, I can move the window, I cannot highlight "Shipwrecks", "Name" or take highlight away from "Your name".
5. Pressing X button does nothing, after a moment, OS asks me if I want to kill the window that is unresponsive.

Final log lines:

```
Launching the client

** (java:6971): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-beTpcQ9Ss7: Connection refused
sty 25, 2018 12:31:48 PM javafx.fxml.FXMLLoader$ValueElement processValue
WARNING: Loading FXML document with JavaFX API of version 8.0.141 by JavaFX runtime of version 8.0.101
sty 25, 2018 12:31:48 PM javafx.fxml.FXMLLoader$ValueElement processValue
WARNING: Loading FXML document with JavaFX API of version 8.0.141 by JavaFX runtime of version 8.0.101
```

**Comment by LIttleAncientForestKami** [comment]
Running: 
```
running
+ echo Running:
Running:
+ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.3 LTS
Release:	16.04
Codename:	xenial
+ inxi -S -xxx
System:    Host: T420s Kernel: 4.4.0-109-generic x86_64 (64 bit gcc: 5.4.0)
           Desktop: Xfce 4.12.3 (Gtk 2.24.28) info: xfce4-panel dm: lightdm
           Distro: Ubuntu 16.04 xenial
+ zsh --version
zsh 5.1.1 (x86_64-ubuntu-linux-gnu)
+ inxi -I -xx
Info:      Processes: 237 Uptime: 1:25 Memory: 2509.4/15928.9MB
           Init: systemd v: 229 runlevel: 5 default: 2 Gcc sys: 5.4.0 alt: 4.8
           Client: Shell (running running in zsh) inxi: 2.2.35 
```
Workaround for accessibility warning (doesn't help with freeze): 
`export NO_AT_BRIDGE=1`

**Comment by LIttleAncientForestKami** [comment]
Problem did not occur after system update. Closing.
